### PR TITLE
Introduce Publicipprefix to swagger

### PR DESCRIPTION
**Pull request by anilingle-ms**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/network/resource-manager/Microsoft.Network/stable/2018-07-01/examples/PublicIpPrefixGet.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      "200" : {
+        "body" : {
+          "name" : "test-ipprefix",
+          "id" : "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/publicIPPrefixes/test-ipprefix",
+          "location" : "westus",
+          "properties" : {
+            "provisioningState" : "Succeeded",
+            "publicIPAddressVersion" : "IPv4",
+            "idleTimeoutInMinutes" : 4,
+            "ipTags" : [
```
Comment (anavinahar) [inline specification/network/resource-manager/Microsoft.Network/stable/2018-07-01/examples/PublicIpPrefixGet.json]: We're exposing tags as part of the prefix? 

Also, should it align with this: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/network/resource-manager/Microsoft.Network/stable/2018-06-01/examples/PublicIpAddressList.json

**Review by anavinahar — APPROVED** [review]

File: specification/network/resource-manager/Microsoft.Network/stable/2018-07-01/publicIpPrefix.json [hunk 2]
```diff
@@ -0,0 +1,426 @@
+{
+    "swagger": "2.0",
+    "info": {
+      "title": "NetworkManagementClient",
+      "description": "The Microsoft Azure Network management API provides a RESTful set of web services that interact with Microsoft Azure Networks service to manage your network resources. The API has entities that capture the relationship between an end user and the Microsoft Azure Networks service.",
+      "version": "2018-06-01"
```
Comment (lmazuel) [inline specification/network/resource-manager/Microsoft.Network/stable/2018-07-01/publicIpPrefix.json]: Incorrect API version

**Review by lmazuel — CHANGES_REQUESTED** [review]
Change ApiVersion, otherwise LGTM. Need ARM feedback.

**Comment by anilingle-ms** [comment]
Fixed. Please Re-Review if needed.

- Anil

**Review by lmazuel — APPROVED** [review]

**Comment by lmazuel** [comment]
@anilingle-ms Actually, your example does not pass validation:
https://travis-ci.org/Azure/azure-rest-api-specs/jobs/410049638

Example, some examples contains `publicIPAllocationMethod` that are not defined in the Swagger. Either add these to the Swagger if they are expected/returned by the server, or remove them from examples.

**Review by lmazuel — CHANGES_REQUESTED** [review]
Examples validation error.

**Review by lmazuel — APPROVED** [review]

**Comment by lmazuel** [comment]
@ravbhatnagar could you check everything is fine fro; an ARM point of view? All PR is passing CI now. Thanks!

**Comment by ravbhatnagar** [comment]
This has been reviewed.

**Pull request merged**